### PR TITLE
Remove .es6 transform and filenames

### PR DESCRIPTION
**Pull request by jonathanKingston**
Related to https://app.asana.com/0/892838074342800/1203977997399511/f

### CODE REVIEW

**Review by sammacbeth — APPROVED** [review]

File: Gruntfile.js [hunk 1]
```diff
@@ -217,10 +217,6 @@ module.exports = function (grunt) {
                 files: watch.contentScope,
                 tasks: ['exec:buildContentScope', 'copy:contentScope']
             },
-            backgroundES6JS: {
-                files: watch.background,
-                tasks: ['browserify:background']
```
Comment (kzar) [inline Gruntfile.js:222]: Could we drop the browserify step above for the UI code too?

**Review by kzar — APPROVED** [review]
LGTM % The comment about removing browserify completely.

File: Gruntfile.js (hunk 1 above)
Comment (jonathanKingston) [reply Gruntfile.js:222]: I just noticed I was missing the other 'browserify:background' I don't think this had a functional change tbh, the bit I removed is just the watcher.

I've pushed to CI to see what explodes; I expect everything :D

Comment (jonathanKingston) [reply Gruntfile.js]: Simple answer is no, we should fast follow with that change though I think.

**Pull request merged**